src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -463,6 +463,7 @@ public async Task<FormulaValue> DecodeResponseAsync(HttpResponseMessage response
                     {
                         Kind = ErrorKind.Network,
                         Severity = ErrorSeverity.Critical,
+                        Span = new Syntax.Span(statusCode, statusCode), // Return HTTP Status code in Span
```

Comment:
>// Return HTTP Status code in Span [](http://example.com/codeflow?start=72&length=34)

?
This seems like a mismatch.  Why is Message not enough?

Reply:
Callers have to parse the message which is not efficient

Reply:
This seems very hacky to have Span sometimes be a range and sometimes be a http error code. 

Do callers need to handle http codes differently?

Reply:
Good idea